### DISCUSSION TITLE
Exchanging Metadata

### DISCUSSION THREAD

**Discussion by willgd-git** [codegreencreative/laravel-samlidp#118] [Q&A]
Just getting started to start using this package as IDP and "24Slides/laravel-saml2" as my package's SP. I trust that SAML2 needs to exchanging some important information such as metadata. However I'm so confused as this package does not require any information like SP's metadata. Could you help me explain about how this package does exchange metadata with SP or does this package not automatically read SP's data to trust it?

**Comment by upwebdesign** [comment]
@WillGD-Hub, this package is a wrapper for [litesaml/lightsaml](https://github.com/litesaml/lightsaml). It provides some helpers to integrate with your Laravel app. Please visit [their documentation](https://docs.litesaml.com/) to learn more about how it works.

**Reply by willgd-git** [reply]
Could you provide me some example of how IDP trust auth request from registered SP?
